### PR TITLE
[Improvement] Support left navigation bar of metadata with dragging and resizing

### PR DESCRIPTION
**Pull request by s7monk**
close: https://github.com/apache/paimon-webui/issues/303

### Purpose

The left navigation bar of metadata supports dragging and resizing.
image

image

### CODE REVIEW

**Review by zhangmo8 — APPROVED** [review]
LGTM

File: paimon-web-ui/src/views/metadata/index.tsx [hunk 1]
```diff
@@ -34,21 +34,32 @@ export default defineComponent({
...
+                {this.currentTable
+                  ? (
+                    <>
+                      <Breadcrumb />
+                      <Tabs />
+                    </>
+                    )
+                  : (
+                    <div class={styles.empty}>
+                      <n-empty description="Select the table. Please"></n-empty>
```
Comment (SteNicholas) [inline paimon-web-ui/src/views/metadata/index.tsx:53]:
```suggestion
                      <n-empty description="Please choose a table."></n-empty>
```

**Review by SteNicholas — COMMENTED** [review]
@s7monk, overall LGTM. I left the minor comment for this pull request.

File: paimon-web-ui/src/views/metadata/index.tsx (hunk 1 above)
Comment (s7monk) [reply paimon-web-ui/src/views/metadata/index.tsx]: done.

**Review by SteNicholas — APPROVED** [review]

**Pull request merged**